code-execution/src/main/java/com/dojo/codeexecution/controller/GameResultController.java

```diff
@@ -0,0 +1,30 @@
+package com.dojo.codeexecution.controller;
+
+import com.dojo.codeexecution.model.TestResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+@RestController
+public class GameResultController {
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @PostMapping(path = "/test/result")
+    public void testResult(@RequestBody TestResult testResult) {
+        String username = testResult.getUsername();
+        int points = testResult.getPoints();
+        final String url = "http://localhost:8080/codenjoy-contest/rest/game/update/" + username + "/score";
```

Comment:
Codenjoy address should be defined in a properties file

Reply:
Denis decided to do the properties file task and in order to add the codeenjoy address, I need to have access to his latest changes.